android-project/app/build.gradle

```diff
@@ -65,4 +65,5 @@ android {
 dependencies {
 	implementation fileTree(include: ['*.jar'], dir: 'libs')
 	implementation 'com.android.support.constraint:constraint-layout:2.0.4'
+	compileOnly 'org.jetbrains:annotations:23.0.0'

```

Comment:
Looking at the imports in DataActivity/ImportActivity this package isn't being used, instead you're referencing android.support.annotation. The current version of that package is [androidx.annotation](https://mvnrepository.com/artifact/androidx.annotation/annotatio, not sure if it's a straightforward migration or something needs to be done to enable compiling for older targets?

Reply:
Looks to work with out it, maybe I should just start ignoring there suggestions.